core/store/src/migrations.rs

```diff
@@ -516,3 +516,15 @@ pub fn migrate_42_to_43(store: &Store) -> anyhow::Result<()> {
     update.commit()?;
     Ok(())
 }
+
+/// Migrates the database from version 44 to 45.
+///
+/// Removes STATE_TRANSITION_START_HEIGHTS key from DBCol::Misc that is no longer needed.
+pub fn migrate_44_to_45(store: &Store) -> anyhow::Result<()> {
```

Comment:
Should I test this migration in some way?

Reply:
Just for my information, do we have a general process on how to do it or is it mostly ad-hoc?

Reply:
as far as I know it is adhoc, but also setup is pretty straightforward, so that makes sense